server/build/webpack.js

```diff
@@ -203,6 +203,24 @@ export default async function createCompiler (dir, { buildId, dev = false, quiet
     })
   }
 
+  function getPackageJsonPath (filePath) {
+    let filePathDirname = dirname(filePath)
+    if (filePathDirname === dir) {
+      return join(dir, 'package.json')
```

Comment:
need to check whether this file exists or probably return `undefined` in this case.